src/actions/find.py

```diff
@@ -0,0 +1,11 @@
+from talon import actions
+from .get_texts import get_texts
+
+
+def run_find_action(targets: dict):
+    """Find text in editor"""
+    texts = get_texts(targets)
+    actions.user.vscode("actions.find")
+    text = "\n".join(texts)
```

Comment:
I'm not sure in what situation we'd run this action with a list target.  In fact don't we check on extension side that it's not a list?  I'd just grab first entry I think, no?

Reply:
No get texts returns array. It's used by homophones and four matters and more. You're thinking about the find all which is an actual proper extension side action. We could always add an argument { ensureSingleTarget: true }

Reply:
Oh wait I got this confused with find in files. But still that one throws an error with multiple targets so I'd argue this one should be consistent and do the same. No?

Reply:
Already fixed :)